components/03-modules/mega-menu/mega-menu.js

```diff
@@ -0,0 +1,74 @@
+class MegaMenu {
+  constructor(element) {
+    this.menu = element;
+    this.focusable = 'a[href], area[href], input:not([disabled]), select:not([disabled]), textarea:not([disabled]), button:not([disabled]), object, embed, *[tabindex], *[contenteditable]';
+    this.focusableChildren = Array.from(this.menu.querySelectorAll(this.focusable));
+    this.firstLevelLinks = Array.from(this.menu.querySelectorAll('.mega-menu__item > .mega-menu__link'));
```

Comment:
Maybe use spread operator `...` instead of `Array.from`
```
this.focusableChildren = [...this.menu.querySelectorAll(this.focusable)];
this.firstLevelLinks = [...this.menu.querySelectorAll('.mega-menu__item > .mega-menu__link')];
```

Reply:
changed